.gitignore

```diff
@@ -15,3 +15,8 @@
 
 # Ignore Byebug command history file.
 .byebug_history
+
+config/database.yml
+db/schema.rb
+.rspec
```

Comment:
> Are you sure you want to ignore `.rspec`?

Yes, 100% sure.

_Dotfiles_ like this are "user config files". Per project config files for rspec should be in `spec/rails_helper.rb`, `spec/spec_helper.rb`, or (the one I prefer, `spec/support/config_xxx.rb`).

All the projects I work with I ignore `.rspec`. Unfortunately not all projects/libraries follow this principle, but I prefer to avoid _dotfiles_ to set configurations per project.

One example (for rspec at least):

I don't like the output of rspec to be `--color=true` but most of the devs do, so for me it is pretty annoying working in a project that has that in the `.rspec`.

my 2¢ :)

Reply:
Okay 👍 Not a big deal. Our convention is to commit `.rspec` as it offers a way to pick sensible project defaults.